packages/libs/client-utils/src/nodejs/deploy-from-directory.ts

```diff
@@ -0,0 +1,194 @@
+import type { ChainId } from '@kadena/types';
+import { readFileSync, readdirSync } from 'fs';
+import { join } from 'path';
+import { deployContract } from '../built-in/deploy-contract';
+import type { IClientConfig } from '../core/utils/helpers';
+import { deployTemplate } from './deploy-template';
+
+interface ITransactionBody {
+  chainId: ChainId;
+  networkId: string;
+  signers: string[];
+  meta: {
+    gasLimit: number;
+    chainId: ChainId;
+    ttl: number;
+    senderAccount: string;
+  };
+  data?: { key: string; value: string };
+  keysets?: { name: string; pred: string; keys: string[] }[];
+}
```

Comment:
Idem

Reply:
can you please use the type deom deploy-contract.ts